File: c/estrutura_condicional/dardo.c
#include <stdio.h>

int main(void)
{
    double distancia1, distancia2, distancia3;
    
    printf("Digite as tres distancias:\n");
    scanf ("%lf\n", &distancia1);
    scanf ("%lf\n", &distancia2);
    scanf ("%lf", &distancia3);
    
    if (distancia1 > distancia2 && distancia1 > distancia3)
    {
        printf("MAIOR DISTANCIA = %.2lf\n", distancia1);
    }
    else if (distancia2 > distancia1 && distancia2 > distancia3)
    {
        printf("MAIOR DISTANCIA = %.2lf\n", distancia2);
    }
    else {
        printf("MAIOR DISTANCIA = %.2lf\n", distancia3);
    }
}

File: python/estrutura_condicional/dardo.py
print("Digite as tres distancias: ")
distancia1 = float(input())
distancia2 = float(input())
distancia3 = float(input())

if distancia1 > distancia2 and distancia1 > distancia3:
    print(f"MAIOR DISTANCIA = {distancia1:.2f}")
elif distancia2 > distancia1 and distancia2 > distancia3:
    print(f"MAIOR DISTANCIA = {distancia2:.2f}")
else:
    print(f"MAIOR DISTANCIA = {distancia3:.2f}")
